plugins/CoreAdminHome/tests/Integration/Commands/ConfigGetTest.php

```diff
@@ -619,27 +619,6 @@ public function testUsingArgsGetArraySettingWithBrackets()
         $this->assertEquals($expectedValue, $resultObj->output);
     }
 
-    public function testUsingOptsCallWithMultipleSectionsReturnsLastSectionOnly()
-    {
-
-        $inputArr = [
-            'command' => self::COMMAND,
-            '--section' => self::TEST_SECTION_2_NAME,
-            '--section' => self::TEST_SECTION_1_NAME,
```

Comment:
This test was actually useless. Providing the same array key two times also has the result that the second one is used and the first one is overwritten. Therefor the tests never passed the section twice. Doing that (by passing an array for it) results in a command failure, as arrays are not accepted, therefor this tests makes no sense and can be removed.

Reply:
Could this test be changed into a string command where we pass the `--section` param twice? That was the original intention of the test, wasn't it?

Reply:
Good point. Haven't thought about that. Will update it.